src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+abstract class Machine {
+

```

Comment:
the same

Reply:
If I removed this line, and run test mvn clean package, after I had error: 'CLASS_DEF' should be separated.......

Reply:
If you'll have such an error, tag me in your private channel